next.config.js

```diff
@@ -1,8 +1,17 @@
+require('dotenv').config();
+
```

Comment:
With next.js we don't need dotenv configs, see here https://nextjs.org/blog/next-9-4#new-environment-variables-support

Reply:
oh, nice. Nextjs never ceases to amaze me!